package.json

```diff
@@ -131,6 +136,7 @@
   },
   "resolutions": {
     "@types/react": "^18.2.0",
-    "esbuild": "^0.18.20"
+    "esbuild": "^0.18.20",
+    "**/gl": "^6.0.2"
```

Comment:
Modules is still using Node 16 at the moment, so I don't think this GL dependency resolution is necessary.